src/main/src/test/java/org/geoserver/util/EntityResolverProviderTest.java

```diff
@@ -49,4 +53,50 @@ public void testAllowListDomains() throws Exception {
         assertTrue(allowed.containsAll(Arrays.asList(AllowListEntityResolver.OGC.split("\\|"))));
         assertTrue(allowed.contains("how2map.com"));
     }
+
+    @Test
+    public void testNotRegEx() throws Exception {
```

Comment:
I'm not sure what this is testing - is "foo*bar" being treated as a regex or is the "\*" just a normal character?

Reply:
Check that the '*' allows all, while "foo*bar" wild card is not supported (ie is a literal).